pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```
@@ -726,7 +726,10 @@ func validateLiveMigration(field *k8sfield.Path, spec *v1.VirtualMachineInstance
 	if spec.EvictionStrategy != nil {
 		evictionStrategy = spec.EvictionStrategy
 	}
-	if !config.LiveMigrationEnabled() &&
+
+	highAvailabilityInfra := isInfrastructureHighlyAvailable(config.GetConfigFromKubeVirtCR())
+	if highAvailabilityInfra &&
```

Why do we want to silently ignore it?